tests/Data/Entries/EntryQueryBuilderTest.php

```diff
@@ -39,7 +39,7 @@ public function entry_is_found_within_all_created_entries_using_entry_facade_wit
     public function entry_is_found_within_all_created_entries_and_select_query_columns_are_set_using_entry_facade_with_find_method_with_columns_param()
     {
         $searchedEntry = $this->createDummyCollectionAndEntries();
-        $columns = ['title'];
+        $columns = ['foo', 'collection'];
```

Comment:
It's irrelevant for this particular test, since it's just about making sure `$columns` are reflected in `->selectedQueryColumns()`, but without selecting a collection it would throw an error. I also changed title to foo to show that the field itself is irrelevant.